imgee/utils.py

```diff
@@ -238,11 +248,12 @@ def get_no_previews_url(size):
 
 
 def get_image_url(image, size=None):
+    from imgee import storage
```

Comment:
Let's find a way to avoid importing inside functions. It should be the last resort.